packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeploySourcePath.test.ts

```diff
@@ -95,33 +95,5 @@ describe('Force Source Deploy Using Sourcepath Option', () => {
       // NOTE: There's currently a limitation on source deploy retrieve that prevents
       // us from mocking SourceClient.tooling.deploy. We'll look into updating the library and this test.
     });
-
-    it('should get the namespace value from sfdx-project.json and deploy using metadata API', async () => {
-      sb.stub(OrgAuthInfo, 'getDefaultUsernameOrAlias').returns(
-        testData.username
-      );
-      sb.stub(OrgAuthInfo, 'getConnection').returns(mockConnection);
-      const getNamespace = sb.stub(SfdxProjectConfig, 'getValue').returns('');
-      const getComponentsStub = sb.stub(
-        RegistryAccess.prototype,
-        'getComponentsFromPath'
-      );
-      const executor = new LibraryDeploySourcePathExecutor();
-      const filePath = path.join(
-        'test',
-        'file',
-        'path',
-        'classes',
-        'apexTest.cls'
-      );
-      const mockToolingDeploy = sb
```

Comment:
do we not need this test anymore?

Reply:
It was failing with the changes and decided not to "fix" it. Seemed like a pointless test since we don't even support setting the namespace when deploying with metadata api, and the test wasn't testing that in the first place.